lib/components/TokenDetailsCard.jsx

```diff
@@ -1,338 +1,252 @@
-import React, { useContext, useEffect, useState } from 'react'
+import React from 'react'
 
 import { Card } from 'lib/components/Card'
 import { InputLabel } from 'lib/components/InputLabel'
 import { TextInputGroup } from 'lib/components/TextInputGroup'
 import { Collapse } from 'lib/components/Collapse'
-import { TokenDropdown } from 'lib/components/TokenDropdown'
 import { PRIZE_POOL_TYPE } from 'lib/constants'
-import { WalletContext } from 'lib/components/WalletContextProvider'
-import { isAddress } from 'lib/utils/isAddress'
-import { fetchTokenChainData } from 'lib/utils/fetchTokenChainData'
-import { fetchYieldSourceChainData } from 'lib/utils/fetchYieldSourceChainData'
+import { Erc20Image } from './Erc20Image'
 
-function isValidTokenData(data) {
-  return data && data.tokenDecimals && data.tokenSymbol && data.tokenName
+const Line = () => <hr className='mt-2 mb-4 sm:mb-8 sm:mt-6 opacity-60 xs:opacity-100' />
+
+const yieldSourceLabel = (prizePool) => {
+  const label = prizePool.yieldProtocol.label.split('-')[1].trim()
```

Comment:
I pasted the address of a cToken `0x5d3a536e4d6dbd6114cc1ead35777bab948e3643` while on Mainnet and it borked

<img width="404" alt="Screen Shot 2021-05-12 at 3 40 07 PM" src="https://user-images.githubusercontent.com/13701258/118034819-68a33900-b338-11eb-9dbe-6311a3656b78.png">

I was trying to test the case for a user making a pool with a cToken, since I thought it would get caught and labelled as a compound prize pool in `fetchPrizePoolType`